src/actions/actions.ts

```diff
@@ -958,10 +958,16 @@ class CommandDeleteToLineEnd extends BaseCommand {
 class CommandChangeToLineEnd extends BaseCommand {
   modes = [ModeName.Normal];
   keys = ["C"];
+  canBePrefixedWithCount = true;
 
   public async exec(position: Position, vimState: VimState): Promise<VimState> {
     return new ChangeOperator().run(vimState, position, position.getLineEnd().getLeft());
   }
+
+  public async execCount(position: Position, vimState: VimState): Promise<VimState> {
```

Comment:
If you override `execCount`, you don't need to keep `exec`. `execCount`'s default behavior is to call `exec`, but if you override it, then `exec` is never called.